utils/HTML.go

```
@@ -1,13 +1,13 @@
-package controllers
+package utils
 
 import (
 	"io/ioutil"
```

golangci-lint (1.62.2)

SA1019: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package [io] or package [os], and those implementations should be preferred in new code. See the specific function documentation for details. 

(staticcheck)